templates/nextflow-aurora-mysql.yaml

```diff
@@ -103,6 +106,18 @@ Resources:
         PasswordLength: 16
       KmsKeyId: !Sub ${AuroraKeyStack.Outputs.Key}
 
+  AuroraMasterSecretRotationSchedule:
```

Comment:
I'm still unable to manually rotate the secret after deploying the stack in `nextflow-dev`. I run into the following error: 

```
Failed to rotate the secret "nextflow-aurora-mysql-MasterSecret". A previous rotation isn't complete. That rotation will be reattempted.
```

![image](https://user-images.githubusercontent.com/740725/229971884-2e56da13-cb8b-4d22-aa02-f788bccc215a.png)

Reply:
@BrunoGrandePhD would you like to give this a try? https://stackoverflow.com/a/50337928/1094247

i can also give it try if you prefer that i debug.

Reply:
I'll look into that tomorrow

Reply:
Indeed, there was a version stuck in `AWSPENDING`. I cleared that stage using the following command. 

```console
aws secretsmanager update-secret-version-stage --secret-id <arn> --version-stage AWSPENDING --remove-from-version-id <version-id>
```

After, I tried rotating the secret manually, and I got a green pop-up message instead of a red one. However, I don't think the Lambda succeeded. So, this might still need some work. I'll defer to Khai.